spec/services/search/base_spec.rb

```diff
@@ -1,13 +1,15 @@
 require "rails_helper"
 
-RSpec.describe Search::Base, type: :service, elasticsearch: true do
+RSpec.describe Search::Base, type: :service do
   let(:document_id) { 123 }
 
   before do
     # Need to use an existing index name to ensure proper data cleanup
     stub_const("#{described_class}::INDEX_NAME", "tags_#{Rails.env}")
     stub_const("#{described_class}::INDEX_ALIAS", "tags_#{Rails.env}_alias")
     stub_const("#{described_class}::MAPPINGS", Search::Tag::MAPPINGS)
+    Search::Tag.refresh_index
```

Comment:
When dealing with [bulk updating operations](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-bulk.html#bulk-update) and [bulk deleting operations](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-delete-by-query.html#docs-delete-by-query) you need to make sure you refresh your index before trying to do a second one otherwise you will end up with conflict errors.